src/details/ArborX_DetailsTreeVisualization.hpp

```diff
@@ -203,7 +154,8 @@ struct TreeVisualization
     template <typename Query>
     KOKKOS_FUNCTION void operator()(Query const &, int index) const
     {
-      _visitor.visit(TreeAccess::getNodePtr(_tree, index), _tree);
+      // FIXME
```

Comment:
Make sure you see that too.  This is a bug.

Reply:
But the code that is being executed is simply wrong. And there is no indication of that. As this is in our visualization, and not core, I thought we should be clear about that.

Reply:
Fixed (I think)